models/bandalarm.go

```diff
@@ -51,11 +51,7 @@ func (b *BandAlarm) FromInternal(internal *models.ModelsBandAlarm) {
 	}
 }
 
-func (b *BandAlarm) ToInternal() *models.ModelsBandAlarm {
-	if b == nil {
```

Comment:
Isn't this nil check needed anymore?

Reply:
No, by changing the receiver to a non-pointer we move this responsibility to the caller and avoid weird pointer-problems that are hard to debug